packages/opentelemetry-exporter-otlp-http/test/node/CollectorMetricExporter.test.ts

```diff
@@ -85,7 +85,7 @@ describe('CollectorMetricExporter - node with json over http', () => {
     const envSource = process.env;
     it('should use url defined in env', () => {
       envSource.OTEL_EXPORTER_OTLP_ENDPOINT = 'http://foo.bar/v1/metrics';
-      const collectorExporter = new CollectorMetricExporter();
+      const collectorExporter = new OTLPMetricExporter();
```

Comment:
Might be worth renaming variable names like this, it's a little confusing since it's an artifact of the old name.

Reply:
I tried to start on that but it was an enormous undertaking. Far bigger than I had expected. I'll do all the exported ones though.